account_consolidation/models/consolidation_profile.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class CompanyConsolidationProfile(models.Model):
+
+    _name = 'company.consolidation.profile'
+    _description = 'Subsidiary consolidation profile'
+    _order = 'sub_company_id'
+    _rec_name = 'sub_company_id'
+
+    @api.model
+    def _default_consolidation_percentage(self):
+        return 100
+
+    @api.model
+    def _default_company(self):
+        return self.env['res.company']._company_default_get()
+
+    company_id = fields.Many2one('res.company',
+                                 default=lambda self: self._default_company(),
```

Comment:
unless this breaks inter-dependencies w/ some module you can drop `_default_company` and just use `self.env['res.company']._company_default_get()` inside lambda.